src/http/request_handler.rs

```diff
@@ -105,11 +105,25 @@ fn create_allowed_headers(headers: &HeaderMap, allowed: &BTreeSet<String>) -> He
     new_headers
 }
 
+pub async fn preflight(app_ctx: Arc<AppContext>, req: Request<Body>) -> Result<Response<Body>> {
+    let _whole_body = hyper::body::aggregate(req).await?;
+    let mut response = Response::builder().status(StatusCode::OK);
+
+    for (key, val) in app_ctx.blueprint.server.response_headers.iter() {
+        response = response.header(key, val)
+    }
```

Comment:
there should be a helper for this defined somewhere use that.

Reply:
using the helper now